spec/03-plugins/18-acl/01-api_spec.lua

```diff
@@ -152,10 +153,114 @@ for _, strategy in helpers.each_strategy() do
           res = assert(admin_client:get("/consumers/alice/acls/" .. acl.group))
           assert.res_status(404, res)
         end)
+        it("retrieves right ACL by group when multiple consumers share the same group name created with POST", function()
+          local res  = assert(admin_client:post("/consumers", {
+            body = {
+              username = "anna",
+            },
+            headers = {
+              ["Content-Type"] = "application/json",
+            },
+          }))
+          local body = assert.res_status(201, res)
+          local _    = cjson.decode(body)
```

Comment:
I think using the following assertion is more explicit, and produces a slightly better error message if something goes wrong:

```suggestion
          assert.response(res).has.jsonbody()
```

The assertion also returns the decoded json table, so it could be used to replace the calls to `cjson.decode` where they are assigned to a named variable too, if you so desired. This is an aesthetic choice and I don't particularly care about that.